src/twisted/internet/_signals.py

```diff
@@ -120,9 +292,8 @@ class _SocketWaker(log.Logger):
 
     disconnected = 0
 
-    def __init__(self, reactor):
+    def __init__(self):
```

Comment:
while we're in here… how much damage would this do
```suggestion
    def __init__(self) -> None:
```